wafer/registration/validators.py

```diff
@@ -0,0 +1,12 @@
+import re
+
+from django.core.exceptions import ValidationError
+
+
+def validate_username(username):
+    """. and .. lead to problematic URLs and static file names"""
+    if re.match(r'^\.+$', username):
+        raise ValidationError(
+            "usernames cannot consist of only .")
```

Comment:
Shouldn't this also mention .. in the error, or say something like 'only .'s' ?

Reply:
Also, shouldn't we block leading .'s entirely? While not as problematic as the directory entries, the creation of arbitrary  hidden filenames can lead to some odd behaviours

Reply:
That sounds reasonable, yes.